src/index.js

```diff
@@ -3,10 +3,17 @@ import ReactDOM from "react-dom";
 
 import "./styles.css";
 import { Game } from "./Game";
+import { BOARD_SIZE_MAX, BOARD_SIZE_MIN } from './constants';
 
-const App = function() {
-  const boardSizeX = 10;
-  const boardSizeY = 10;
+const App = function () {
+  // eslint-disable-next-line
+  let size = parseInt(new URLSearchParams(location.search).get('size'), 10);
+  if (isNaN(size)) size = BOARD_SIZE_MAX;
```

Comment:
a simpler way is

```js
let size = parseInt(paramSize, 10) || BOARD_SIZE_MAX;
```

look at the comment before this one.

Reply:
Done.  However, one test is outdated: checking 10&times;10 board size at the beginning.